plugins/modules/ovirt_vm.py

```diff
@@ -1414,8 +1414,24 @@ def __get_template_with_version(self):
         if self._is_new:
             if self.param('template'):
                 clusters_service = self._connection.system_service().clusters_service()
+                data_centers_service = self._connection.system_service().data_centers_service()
                 cluster = search_by_name(clusters_service, self.param('cluster'))
-                data_center = self._connection.follow_link(cluster.data_center)
+                # For default DC/Cluster we use the hardcoded name 'Default' even that name
+                # can be modified, we have no other indication to know that.
+                # The API does not return a href for the data_center of the default cluster
```

Comment:
Why API behaves differently for the default DC? What's the reason to not return the href to the default cluster same as for the other clusters?

@oliel